README.md

```diff
@@ -106,6 +106,15 @@ The response will be like:
 }
 ```
 
+## Settings
+You can set default key for Django's non field errors key (it is `"__all__"`) by adding this to your Django settings:
+
+```
+HIPO_DRF_EXCEPTIONS_SETTINGS = {
+    "DJANGO_NON_FIELD_ERRORS_KEY": "field_free_errors"  # by default it's "non_field_errors" to be consistent with DRF.
+}
+```
+
```

Comment:
I am not sure if this should be treated as a breaking change. Seems to me that this is a bug. Not sure any of the clients handled this bug before. Even though they handled when we switched to non_field_errors keys which is fallback everything should be fine (in theorie :)).

What do you think @yigitguler ?

Reply:
I have though the same while implementing this but then I have realized that sometimes we use `__all__` key in our tests. If we update this package on some project, tests might fail. In this aspect this might be a breaking change, otherwise I totally agree with you.